andela/ah-jarvis-frontend
# [ch #160506498] Add logout button

#### What does this PR do?
- Adds logout button
- Adds default image if user has not added profile image

#### Description of Task to be completed?
- The user will be able to logout

#### How should this be manually tested?
1. Clone this repo and branch git clone `git@github.com:andela/ah-jarvis-frontend.git -b ch-add-logout-button-160506498`
2. Open the project in your prefered text editor
3. Run the code below in your terminal
```
yarn install

yarn start
```
You should be redirected to your browser
4. Navigate to this url: `http://localhost:3000/signin`
5. Login
6. Click logout from the dropdown on the top right.

#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?

[#160506498 ](https://www.pivotaltracker.com/story/show/160506498) Add a logout route and link on the dashboard

#### Screenshots (if appropriate)
<img width="408" alt="screen shot 2018-09-14 at 13 21 44" src="https://user-images.githubusercontent.com/18376530/45544866-3789a800-b821-11e8-9138-9dd6d509acae.png">
